app/views/spotlight/translations/_browse_categories.html.erb

```diff
@@ -0,0 +1,66 @@
+<div role="tabpanel" class="tab-pane" id="browse">
+  <%= bootstrap_form_for current_exhibit, url: spotlight.exhibit_translations_path(current_exhibit), layout: :horizontal do |f| %>
```

Comment:
Would you mind adding a hidden field here for the language? https://github.com/projectblacklight/spotlight/pull/1940/files#diff-f6fd4ed34036df67d84f28e10b98eac8R3

This way the page should correctly reload this language tab after save

Reply:
Good catch, will add that.